akka-cluster-sharding/src/main/scala/akka/cluster/sharding/ShardRegion.scala

```diff
@@ -548,6 +544,17 @@ private[akka] class ShardRegion(
     gracefulShutdownProgress.trySuccess(Done)
   }
 
+  private def logPassivateIdleEntities(): Unit = {
+    if (settings.shouldPassivateIdleEntities)
+      log.info(
+        "{}: Idle entities will be passivated after [{}]",
+        typeName,
+        PrettyDuration.format(settings.passivateIdleEntityAfter))
+
+    if (settings.rememberEntities)
+      log.warning("Idle entities will not be passivated because 'rememberEntities' is enabled.")
```

Comment:
I'm not sure that should be a warning since it's expected and documented behavior. Could be debug.

Reply:
Info is probably better.

Reply:
Oh debug yes.